psydac/core/bsplines.py

```diff
@@ -568,11 +572,10 @@ def make_knots(breaks, degree, periodic, multiplicity=1, out=None):
     assert all( np.diff(breaks) > 0 )
     assert degree > 0
     assert 1 <= multiplicity and multiplicity <= degree + 1
-
+    multiplicity = int(multiplicity)
```

Comment:
It would fail because sometimes, the multiplicity comes from a numpy array which transforms python `int` into `np.int64` which for some reason aren't accepted by pyccel when they arguably should.

Reply:
`int` and `np.int64` are not equivalent in Pyccel because `int` is stored as a 32 bit integer on Windows machines 🤣

Reply:
In any case, this conversion should allow the code to work on windows.